packages/constellation/src/components/Compositions/Logo/Logo.stories.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import * as Icons from '../../Base/Icon'
+import { Logo, LogoComponent, LogoProps } from '.'
+
+export default {
+  argTypes: {
+    circle: {
+      control: { type: 'boolean' },
+    },
+    primaryContent: { control: { disable: true } },
+    primaryIcon: {
+      control: { type: 'select' },
+      options: Object.keys(Icons),
+    },
+    primaryIconColor: {
+      control: { type: 'color' },
+    },
+    primaryImgSrc: {
+      control: { type: 'text' },
+    },
+    primaryText: {
+      control: { type: 'text' },
+    },
+    secondaryContent: { control: { disable: true } },
+    secondaryIcon: {
+      control: { type: 'select' },
+      options: Object.keys(Icons),
+    },
+    secondaryIconColor: {
+      control: { type: 'color' },
+    },
+    secondaryImgSrc: {
+      control: { type: 'text' },
+    },
+    secondaryText: {
+      control: { type: 'text' },
+    },
```

Comment:
I realize this makes it easier to modify the different aspects of the Logo within Storybook, but do you think this someone might confuse these as the required props (as opposed to the actual `primaryContent` and `secondaryContent` props?

I wonder if there is a way to call this out in the story or in a separate intro?